UI-UX Design/README.md

```diff
@@ -1,6 +1,6 @@
 # UI & UX Design 🎨
 
-## Table of Contents
+## Navigator
```

Comment:
```suggestion
## Table of Contents
```

Reply:
yes i changed it and sorry for the delay